day_two_guide/topics/proc_scaling-etcd-ansible.adoc

```diff
@@ -0,0 +1,78 @@
+////
+scaling etcd using Ansible
+
+Module included in the following assemblies:
+
+* admin_guide/assembly_replace-etcd-member.adoc
+* admin_guide/assembly_restoring-cluster.adoc
+* admin_guide/assembly_replace-master-host.adoc
+////
+
+[id='adding-etcd-host-ansible_{context}']
+= Adding a new etcd host using Ansible
+
+[discrete]
+== Procedure
+
+. In the Ansible inventory file, create a new group named `[new_etcd]`
+and add the new host. Then, add the `new_etcd` group as a child of the `[OSEv3]`
+group:
++
+----
+[OSEv3:children]
+masters
+nodes
+etcd
+<new_etcd>
```

Comment:
Should we remove the angle brackets ?

Reply:
I think they were added to emphasize the added lines, so I'm adding an annotation instead.